.github/workflows/lint-scss.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint SCSS stylesheets
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup NPM
+      uses: actions/setup-node@v3
+
+    - run: npm install --production=false
```

Comment:
We may want to consider adding Action Cache in the future to cache NPM dependencies. At this time, it doesn't appear the install process take very long because there aren't many dependencies so this is more "for later if things get slow" than a change we should make now.

https://github.com/actions/cache

Example for our Rails apps where it loads a cached copy if there have been no changes to the lock file/ruby version combination which makes "Installing" go super fast as it just uses the cached copies.

https://github.com/MITLibraries/.github/blob/main/.github/workflows/ruby-shared-ci.yml#L13-L18

Reply:
Thanks for the callout here. I'm fine with this being in our pocket in case things slow down unacceptably.